src/components/task-plan/reading.cjsx

```diff
@@ -38,7 +40,22 @@ SelectTopics = React.createClass
     return
 
   render: ->
-    <BS.Button onClick={@handleToggle} bsStyle="primary">Edit Readings</BS.Button>
+    if (@props.selected.length and TocStore.isLoaded())
```

Comment:
It's OK if the button does not display; I was just concerned about the async state between loading the ToC (or plan) and letting the user edit it.